doc/code/overview.rst

```diff
@@ -0,0 +1,91 @@
+
+Other modules
+-------------
+
+The following modules are contain lower-level functionality, which is more relevant for developers:
+
++-------------------------------+--------------------------------------------------------------------------------------+
+| :ref:`pennylane.about         |  Print basic information about PennyLane.                                            |
+| <about>`                      |                                                                                      |
++-------------------------------+--------------------------------------------------------------------------------------+
+| :ref:`pennylane._device       |  Defines the abstract :class:`Device` class,                                         |
+| <device>`                     |  which plugins inherit from.                                                         |
++-------------------------------+--------------------------------------------------------------------------------------+
+| :ref:`pennylane.interfaces    |  Contains the quantum nodes for different interfaces.                                |
+| <interfaces>`                 |                                                                                      |
++-------------------------------+--------------------------------------------------------------------------------------+
+| :ref:`pennylane.operation     |  Defines the abstract :class:`Operation` class from                                  |
+| <operation>`                  |  which all operations, such as gates and observables, are derived.                   |
++-------------------------------+--------------------------------------------------------------------------------------+
+| :ref:`pennylane.plugins       |  Contains the default plugins ``default.qubit`` and ``default.gaussian``.            |
+| <plugins>`                    |                                                                                      |
++-------------------------------+--------------------------------------------------------------------------------------+
+| :ref:`pennylane.utils         |  Contains a collection of utility functions.                                         |
+| <utils>`                      |                                                                                      |
++-------------------------------+--------------------------------------------------------------------------------------+
+| :ref:`pennylane.variable      |  Defines the :class:`Variable` class for tunable                                     |
+| <variable>`                   |  parameters.                                                                         |
++-------------------------------+--------------------------------------------------------------------------------------+
+
+
+.. note::
+
+    If you want to write a plugin that connects PennyLane to hardware
+    and simulator backends, visit the :ref:`Plugin API <plugin_overview>`.
```

Comment:
If you want to write a plugin that connects PennyLane to hardware and simulator backends, visit the :ref:`Plugin API <plugin_overview>` page.

Reply:
```suggestion
    and simulator backends, visit the :ref:`Plugin API <plugin_overview>` page.
```